.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# At least one of the code owners below will be required on each PR:
+
+* @smmatte @rectified95 @rachel-slater @jackbk @turalf @markAtMicrosoft @matthidinger
```

Comment:
Will this affect the other people outside the internship working on this?  If so we can specify just the ./private-templates-service folder

Reply:
The codeowners file is ONLY used when specifically assigned to a particular branch.  For example we will assign to the interns "dev" branch a requirement that at least one codeowner must approve a PR before it can merged into dev.

Reply:
It will not affect any other branch or PR unless so designated.